keras_cv/bounding_box/converters_test.py

```diff
@@ -47,11 +55,26 @@
     "rel_yxyx": rel_yxyx_box,
 }
 
+boxes_ragged_images = {
+    "xyxy": xyxy_box,
+    "center_xywh": center_xywh_box,
+    "rel_xywh": rel_xywh_box * tf.constant([[10.0], [20.0]]),
```

Comment:
Can we actually hard code these values in place instead of generating them on the fly?

This makes it easier for us to visually verify that all of the bounding boxes have the same meaning.

Perhaps leave a code comment as well.

Reply:
Do you want to explicitly "duplicate/fork" all the `rel_` variables?

Reply:
I've explicitly forked the rel vars (reopen this if this is not what you meant).